From 7b5baa9e8499564f5538a49b3d50a2f0996fe3a9 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Thu, 25 Aug 2016 01:33:32 -0400
Subject: [PATCH 01/62] Remove unnecessary --noverifyssl for http/ftp protocol

We set --noverifyssl by default, but it makes sense for https only
---
 pyanaconda/install.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/pyanaconda/install.py b/pyanaconda/install.py
index 62ec23d..8dd872e 100644
--- a/pyanaconda/install.py
+++ b/pyanaconda/install.py
@@ -49,6 +49,10 @@ def _writeKS(ksdata):
                ksdata.partition.dataList() + ksdata.raid.dataList():
         obj.passphrase = ""
 
+    # The true of '--noverifyssl' makes sense for https only
+    if ksdata.method.method == "url" and not ksdata.method.url.startswith("https://"):
+        ksdata.method.noverifyssl = False
+
     with open(path, "w") as f:
         f.write(str(ksdata))
 
-- 
2.7.4

